configs/keycloak/realms/ozone-realm.json

```diff
@@ -501,6 +501,41 @@
     ]
   },
   "users": [
+    {
```

Comment:
Could you add `gender` attribute similar to how we have done [here](https://github.com/ozone-his/ozone/pull/170/files)